data/pathfinder/paizo/roleplaying_game/ultimate_combat/uc_equip_arms_armor.lst

```diff
@@ -123,28 +123,28 @@ Wushu Dart (5)				PROFICIENCY:WEAPON|Wushu Dart					TYPE:Weapon.Resizable.Easter
+Pistol															PROFICIENCY:WEAPON|Pistol								TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=1				COST:1000	WT:4										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d8					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.138	SPROP:Misfire 1 (5 ft)
+Pistol (Coat)							OUTPUTNAME:Coat Pistol				PROFICIENCY:WEAPON|Pistol (Coat)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=1				COST:750	WT:1										CRITMULT:x3	CRITRANGE:1							DAMAGE:1d4					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1 (5 ft.)
+Pistol (Dagger)							OUTPUTNAME:Dagger Pistol			PROFICIENCY:WEAPON|Pistol (Dagger)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol.Double		ALTTYPE:Weapon.Resizable.Melee.Light.Finesseable.Simple.Standard.Piercing.Slashing.Dagger.BladeLight	CONTAINS:1|FirearmBullet=1				COST:740	WT:1										CRITMULT:x3	CRITRANGE:1							DAMAGE:1d4					EQMOD:STEEL										ALTEQMOD:STEEL	WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1 (5 ft.)
+Pistol (Double-Barreled)					OUTPUTNAME:Double-Barreled Pistol		PROFICIENCY:WEAPON|Pistol (Double-Barreled)					TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=2				COST:1750	WT:5										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d8					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1-2 (5 ft.)
+Pistol (Dragon)							OUTPUTNAME:Dragon Pistol			PROFICIENCY:WEAPON|Pistol (Dragon)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol.Scatter																			CONTAINS:1|FirearmBullet=1|FirearmPellet=1	COST:1000	WT:3										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d6					EQMOD:FIREARM_SCATTER.STEEL										WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1-2 (5 ft.); Scatter 15' cone
+Pistol (Sword Cane)						OUTPUTNAME:Sword Cane Pistol			PROFICIENCY:WEAPON|Pistol (Sword Cane)						TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol.Double		ALTTYPE:Weapon.Resizable.Melee.OneHanded.Martial.Standard.Piercing.Sword					CONTAINS:1|FirearmBullet=1				COST:775	WT:1										CRITMULT:x3	CRITRANGE:1							DAMAGE:1d4					EQMOD:STEEL										ALTEQMOD:STEEL	WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1 (5 ft.)
```

Comment:
Look at Sword Cain


On 7/8/2016 11:39 PM, Regan Anderson wrote:
>
> In
> data/pathfinder/paizo/roleplaying_game/ultimate_combat/uc_equip_arms_armor.lst
> <https://github.com/PCGen/pcgen/pull/1363#discussion_r70163486>:
>
> > +Blunderbuss															PROFICIENCY:WEAPON|Blunderbuss							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Blunderbuss.Scatter																		CONTAINS:1|FirearmBullet=1|FirearmPellet=1	COST:2000	WT:8										CRITMULT:x2	CRITRANGE:1							DAMAGE:1d8					EQMOD:FIREARM_SCATTER.STEEL										WIELD:TwoHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.137	SPROP:Misfire 1-2 (10 ft); Scatter 15' cone
> > +Buckler Gun															PROFICIENCY:WEAPON|Buckler Gun							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=1				COST:750	WT:6										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d6					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.137	SPROP:Misfire 1 (5 ft.)
> > +Culverin															PROFICIENCY:WEAPON|Culverin								TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Culverin.Scatter																			CONTAINS:1|FirearmBullet=1				COST:4000	WT:40										CRITMULT:x4	CRITRANGE:1							DAMAGE:2d8					EQMOD:FIREARM_SCATTER.STEEL										WIELD:TwoHandedOnly	RANGE:30				SIZE:M	SOURCEPAGE:p.137	SPROP:Misfire 1 (10 ft.)	SPROP:Firing a culverin without support (such as a wall, a window, or a stand) imparts a -4 penalty on the attack rolls, and the wielder is knocked prone. A culverin uses 4 doses of black powder and grapeshot.
> > +Double Hackbut														PROFICIENCY:WEAPON|Double Hackbut							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Hackbut																				CONTAINS:1|FirearmBullet=1				COST:4000	WT:18										CRITMULT:x4	CRITRANGE:1							DAMAGE:2d12					EQMOD:STEEL													WIELD:TwoHandedOnly	RANGE:50				SIZE:M	SOURCEPAGE:p.137	SPROP:Misfire 1-2 (5 ft.)
> > +Fire Lance															PROFICIENCY:WEAPON|Fire Lance								TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.FireLance																				CONTAINS:1|Javelin=1					COST:25	WT:4										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d6					EQMOD:STEEL													WIELD:TwoHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.137	SPROP:Misfire 1-4 (5 ft.)
> > +Musket															PROFICIENCY:WEAPON|Musket								TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Musket																					CONTAINS:1|FirearmBullet=1				COST:1500	WT:9										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d12					EQMOD:STEEL													WIELD:TwoHandedOnly	RANGE:40				SIZE:M	SOURCEPAGE:p.137	SPROP:Misfire 1-2 (5 ft)
> > +Musket (Axe)							OUTPUTNAME:Axe Musket				PROFICIENCY:WEAPON|Musket (Axe)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Musket.Double		ALTTYPE:Weapon.Resizable.Melee.OneHanded.Martial.Standard.Slashing.Axe						CONTAINS:1|FirearmBullet=1				COST:1600	WT:6										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d8					EQMOD:STEEL										ALTEQMOD:STEEL	WIELD:TwoHandedOnly	RANGE:30				SIZE:M	SOURCEPAGE:p.137	SPROP:Misfire 1-2 (5 ft.)
> > +Musket (Double-Barreled)					OUTPUTNAME:Double-Barreled Musket		PROFICIENCY:WEAPON|Musket (Double-Barreled)					TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Musket																					CONTAINS:1|FirearmBullet=2				COST:2500	WT:11										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d12					EQMOD:STEEL													WIELD:TwoHandedOnly	RANGE:40				SIZE:M	SOURCEPAGE:p.138	SPROP:Misfire 1-3 (5 ft.)
> > +Musket (Warhammer)						OUTPUTNAME:Warhammer Musket			PROFICIENCY:WEAPON|Musket (Warhammer)						TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Musket.Double		ALTTYPE:Weapon.Resizable.Melee.OneHanded.Martial.Standard.Bludgeoning.Hammer					CONTAINS:1|FirearmBullet=1				COST:1600	WT:6										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d8					EQMOD:STEEL										ALTEQMOD:STEEL	WIELD:TwoHandedOnly	RANGE:30				SIZE:M	SOURCEPAGE:p.138	SPROP:Misfire 1-2 (5 ft.)
> > +Pepperbox															PROFICIENCY:WEAPON|Pepperbox								TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=6				COST:3000	WT:5										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d8					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.138	SPROP:Misfire 1-2 (5 ft.)
> > +Pistol															PROFICIENCY:WEAPON|Pistol								TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=1				COST:1000	WT:4										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d8					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.138	SPROP:Misfire 1 (5 ft)
> > +Pistol (Coat)							OUTPUTNAME:Coat Pistol				PROFICIENCY:WEAPON|Pistol (Coat)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=1				COST:750	WT:1										CRITMULT:x3	CRITRANGE:1							DAMAGE:1d4					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1 (5 ft.)
> > +Pistol (Dagger)							OUTPUTNAME:Dagger Pistol			PROFICIENCY:WEAPON|Pistol (Dagger)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol.Double		ALTTYPE:Weapon.Resizable.Melee.Light.Finesseable.Simple.Standard.Piercing.Slashing.Dagger.BladeLight	CONTAINS:1|FirearmBullet=1				COST:740	WT:1										CRITMULT:x3	CRITRANGE:1							DAMAGE:1d4					EQMOD:STEEL										ALTEQMOD:STEEL	WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1 (5 ft.)
> > +Pistol (Double-Barreled)					OUTPUTNAME:Double-Barreled Pistol		PROFICIENCY:WEAPON|Pistol (Double-Barreled)					TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=2				COST:1750	WT:5										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d8					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1-2 (5 ft.)
> > +Pistol (Dragon)							OUTPUTNAME:Dragon Pistol			PROFICIENCY:WEAPON|Pistol (Dragon)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol.Scatter																			CONTAINS:1|FirearmBullet=1|FirearmPellet=1	COST:1000	WT:3										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d6					EQMOD:FIREARM_SCATTER.STEEL										WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1-2 (5 ft.); Scatter 15' cone
> > +Pistol (Sword Cane)						OUTPUTNAME:Sword Cane Pistol			PROFICIENCY:WEAPON|Pistol (Sword Cane)						TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol.Double		ALTTYPE:Weapon.Resizable.Melee.OneHanded.Martial.Standard.Piercing.Sword					CONTAINS:1|FirearmBullet=1				COST:775	WT:1										CRITMULT:x3	CRITRANGE:1							DAMAGE:1d4					EQMOD:STEEL										ALTEQMOD:STEEL	WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1 (5 ft.)
>
> Which one? I didn't change the SPROP on the culverin. I copied the
> other existing misfire or scatter cone lines for any I changed.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCGen/pcgen/pull/1363/files/a39690978daedbe3a4d15b1969fb761c49c806ff#r70163486>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACocw77Wc9w6KP9vrOhhYdPkZE3YZrd9ks5qT0I_gaJpZM4JIjYZ>.
>